plenum/server/node.py

```diff
@@ -1160,13 +1170,6 @@ def _ask_for_ledger_status(self, node_name: str, ledger_id):
     def send_ledger_status_to_newly_connected_node(self, node_name):
         self.sendLedgerStatus(node_name,
                               self.ledgerManager.ledger_sync_order[0])
-        # Send the domain ledger status only when it has discovered enough
-        # peers otherwise very few peers will know that this node is lagging
-        # behind and it will not receive sufficient consistency proofs to
-        # verify the exact state of the ledger.
-        if Mode.is_done_discovering(self.mode):
```

Comment:
When catching up, only `LedgerStatuses` for the pool ledger may be gotten without requesting them (in case the node has just started), `LedgerStatuses` for all the other ledgers are always requested.